### DISCUSSION TITLE
Failure when receiving data from peer...

### DISCUSSION THREAD

**Discussion by kapral18** [LazyVim/LazyVim#1704] [Q&A] [answered]
I suddenly keep seeing these after hitting `U` in lazy dashboard.

![CleanShot 2023-10-13 at 19 07 34@2x](https://github.com/LazyVim/LazyVim/assets/1625373/8aa860ca-c2ae-413b-a2c5-2738994bb0ce)

Every time it's a different set of packages failed

Im on latest config https://github.com/kapral18/config.nvim

**Comment by folke** [comment, marked as answer]
That's just github acting up. Can happen sometimes. Nothing I can do about that.

If it keeps happening for you, then set concurrency for lazy.nvim to a low number

**Reply by kapral18** [reply]
keeps failing even with concurrency = 1, continuously

Somehow with less concurrency fails even more

**Reply by kapral18** [reply]
ok closing this. Thanks. Opened this instead https://github.com/orgs/community/discussions/70049
